tck/features/CreateAcceptance.feature

```diff
@@ -18,7 +18,7 @@
 Feature: CreateAcceptance
 
   Scenario: Create a single node with multiple labels
-    Given any graph
+    Given an empty graph
```

Comment:
Right, this is necessary for the `labels` metric, as we don't know if the labels were in the graph or not.